### ISSUE TITLE
How to override home dir using env variable

### ISSUE THREAD

**Issue by kenorb** [FunctionX/fx-core#6]
According to `public/config.toml` in `testnet` branch we can override the home folder by:

```
# NOTE: Any path below can be absolute (e.g. "/var/myawesomeapp/data") or
# relative to the home directory (e.g. "data"). The home directory is
# "$HOME/.tendermint" by default, but could be changed via $TMHOME env variable
# or --home cmd flag.
```

However I've set `$TMHOME` and `fxcored` is still using the default directory (`~/.fxcore`).

Is there any way of changing the default home dir? Example variable can be called: `$FXHOME`.

**Comment by RichardNgKongYeow** [comment]
i have escalated the issue. there will be a fix soon. in the mean time you can try this command:
fxcored init xx --home fileName

**Comment by kenorb** [comment]
Is this already fixed in f42707fab472d18574f57fc4f0ff33e1a9a566dd?

**Comment by zakir-code** [comment]
yes
